modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java

```diff
@@ -865,7 +865,8 @@ private HttpServletRequest _getDDMStructureHttpServletRequest(
 	}
 
 	private Date _getExpirationDate(
-			UploadPortletRequest uploadPortletRequest, TimeZone timeZone)
+			UploadPortletRequest uploadPortletRequest, boolean addDynamic,
```

Comment:
Rename `addDynamic` to something that represents better its function, like "neverExpireDefaultValue" or even "neverExpire".

Reply:
I'm the very best naming, I know :D